cmd/kube-proxy/app/options.go

```diff
@@ -121,13 +133,13 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 		"This parameter is ignored if a config file is specified by --config.")
 
 	fs.Int32Var(o.config.IPTables.MasqueradeBit, "iptables-masquerade-bit", ptr.Deref(o.config.IPTables.MasqueradeBit, 14), "If using the iptables or ipvs proxy mode, the bit of the fwmark space to mark packets requiring SNAT with.  Must be within the range [0, 31].")
-	fs.BoolVar(&o.config.IPTables.MasqueradeAll, "masquerade-all", o.config.IPTables.MasqueradeAll, "If using the iptables or ipvs proxy mode, SNAT all traffic sent via Service cluster IPs. This may be required with some CNI plugins.")
+	fs.BoolVar(&o.config.Linux.MasqueradeAll, "masquerade-all", o.config.Linux.MasqueradeAll, "If using the linux proxy mode, SNAT all traffic sent via Service cluster IPs. This may be required with some CNI plugins.")
```

Comment:
"linux" isn't a "proxy mode". I'd probably say: `"SNAT all traffic sent via Service cluster IPs. This may be required with some CNI plugins. Only supported on Linux."`